cypress/integration/text_duplicator_spec.js

```diff
@@ -1,15 +1,13 @@
-describe("text duplicator", () => {
-    it("Valid path exists to text duplicator", () => {
-      cy.visit("./textDuplicator.html");
```

Comment:
Similar question to https://github.com/joinpursuit/combining-html-and-javascript-plus-dom/pull/60: should these tests start with a `cy.visit`? I'll assume yes unless told not by the end of this week. 😄

Reply:
The answer is yes, thank you for flagging this.
I think in the other repo, all of the tests visited the same page (`index.html` or whatever) so we dried that up by moving `cy.visit` to a `beforeEach()` in the commands.js file. For this lab, they are working on a few different html files so that pattern wouldn't work. Instead, each spec will need to start with a `cy.visit`.
Let me know if this does not make sense! :)